tests/UI/campaigns/functional/BO/02_orders/01_orders/viewAndEditOrder/07_messagesBlock.js

```diff
@@ -0,0 +1,394 @@
+      await expect(isVisible).to.be.true;
+    });
+  });
+
+  // 3 - Check message in FO
+  describe('Check message in FO', async () => {
+    it('should view my shop', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToFoToCheckStatus', baseContext);
+
+      // Click on view my shop
+      page = await viewOrderPage.viewMyShop(page);
+
+      // Change FO language
+      await foHomePage.changeLanguage(page, 'en');
+
+      const isHomePage = await foHomePage.isHomePage(page);
+      await expect(isHomePage, 'Fail to open FO home page').to.be.true;
+    });
+
+    it('should go to login page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToLoginPageFoToCheckStatus', baseContext);
+
+      await foHomePage.goToLoginPage(page);
+      const pageTitle = await foLoginPage.getPageTitle(page);
+      await expect(pageTitle, 'Fail to open FO login page').to.contains(foLoginPage.pageTitle);
+    });
+
+    it('should sign in with default customer', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'signInFo1', baseContext);
+
+      await foLoginPage.customerLogin(page, DefaultCustomer);
+      const isCustomerConnected = await foLoginPage.isCustomerConnected(page);
+      await expect(isCustomerConnected, 'Customer is not connected').to.be.true;
+    });
+
+    it('should go to orders history page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToOrderHistoryPage1', baseContext);
+
+      await foHomePage.goToMyAccountPage(page);
+
+      await foMyAccountPage.goToHistoryAndDetailsPage(page);
+
+      const pageTitle = await foOrderHistoryPage.getPageTitle(page);
+      await expect(pageTitle, 'Fail to open order history page').to.contains(foOrderHistoryPage.pageTitle);
+    });
+
+    it('should go to the first order in the list and check order message', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkOrderMessageBlock1', baseContext);
+
+      await foOrderHistoryPage.goToDetailsPage(page, 1);
+
+      const isBoxMessagesVisible = await foOrderHistoryPage.isBoxMessagesSectionVisible(page);
+      await expect(isBoxMessagesVisible, 'Box messages is not visible!').to.be.true;
+
+      const isMessageRowVisible = await foOrderHistoryPage.isMessageRowVisible(page);
+      await expect(isMessageRowVisible, 'Message is not visible!').to.be.true;
+    });
+
+    // Issue https://github.com/PrestaShop/PrestaShop/issues/26532
+    /* it('should check the message text', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkOrderMessageBlock2', baseContext);
+
+      const message = await foOrderHistoryPage.getMessageRow(page);
+      await expect(message)
+        .to.contain(today)
+        .and.to.contain(`${DefaultEmployee.firstName} ${DefaultEmployee.lastName}`)
+        .and.to.contain(textMessage);
+    }); */
```

Comment:
```suggestion
    // Issue https://github.com/PrestaShop/PrestaShop/issues/26532
    it.skip('should check the message text', async function () {
      await testContext.addContextItem(this, 'testIdentifier', 'checkOrderMessageBlock2', baseContext);

      const message = await foOrderHistoryPage.getMessageRow(page);
      await expect(message)
        .to.contain(today)
        .and.to.contain(`${DefaultEmployee.firstName} ${DefaultEmployee.lastName}`)
        .and.to.contain(textMessage);
    });
```

Like that we can always track skipped steps on reports

Reply:
Thx done :+1: